renderer/index.js

```diff
@@ -251,6 +251,35 @@ function addTorrentEvents (torrent) {
   }
 }
 
+function restoreSession () {
+  cfg.read(function (err, data) {
+    if (err) console.error(err)
+    state.saved = data
+    if (!state.saved.torrents) state.saved.torrents = []
```

Comment:
If we detect it's an empty object, can't we just not do `state.saved = data`, since `state.saved.torrents` is already set to  an initial value in state.js?